frontend/src/app/search.tsx

```diff
@@ -32,7 +32,8 @@ export default class Search extends Component {
     pageLimit: 10,
     redirect: false,
     redirectLocation: '',
-    sortKey: ''
+    showOptions: true,
```

Comment:
showOptions? options related to what? maybe we need to rename this also.

Reply:
Thanks for pointing this out Joseph.
The drop down options are hidden at the bottom of the page, so had to do this check.
I could change the name to 'showDropdownOptions' instead. Does that variable name sound good?

Reply:
yes